var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -208,6 +209,12 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("kokkos-nvcc-wrapper@develop", when="@develop+wrapper")
     depends_on("kokkos-nvcc-wrapper@master", when="@master+wrapper")
     conflicts("+wrapper", when="~cuda")
+    conflicts("+wrapper", when="+compile_as_cmake_language")
+    conflicts(
+        "+compile_as_cmake_language",
+        when="@:3.5",
+        msg="CMake language support requires Kokkos >= 3.6",
+    )
```

Comment:
```suggestion
```

If the variant is conditionally defined.

Reply:
Removed.